examples/ccxt.pro/php/many-exchanges-synchronously.php

```diff
@@ -1,27 +0,0 @@
-<?php
```

Comment:
@ttodua I've read the PR description but I'd argue we should not remove these examples, we have tons of them for the other languages and some of them are similar to each other, as long as they work properly there's no reason to ditch them

Reply:
I think that having tons of similar examples are actually bad, but ofc, i'll revert them back.

Reply:
I've reverted those files too, this PR is also ready.